travis.sh

```diff
@@ -67,6 +67,59 @@ build_java_oracle8() {
   build_java
 }
 
+build_android() {
+  # Set up Gradle, then update to latest version.
+  # http://www.tothenew.com/blog/gradle-installation-in-ubuntu/
+  sudo apt-get -y install python-software-properties # for apt-add-repository
+  yes | sudo add-apt-repository ppa:cwchien/gradle
+  sudo apt-get update
+  sudo apt-get -y install gradle-2.11
+  gradle -v
+  # Set up the Android SDK.
+  # https://gist.github.com/wenzhixin/43cf3ce909c24948c6e7
+  # We can't use the default Android Travis runtime, because we are a
+  # multi-language project.
+  # Need 32-bit versions of the libraries.
+  # http://stackoverflow.com/a/19524010/101923
+  sudo dpkg --add-architecture i386
+  sudo apt-get -qqy update
+  # adb
+  sudo apt-get -y install libc6:i386 libstdc++6:i386
+  # aapt
+  sudo apt-get -y install zlib1g:i386
+  wget http://dl.google.com/android/android-sdk_r24.4.1-linux.tgz
+  tar xzf android-sdk_r24.4.1-linux.tgz
+  export ANDROID_HOME=$PWD/android-sdk-linux
+  export PATH="${PATH}:${ANDROID_HOME}/tools:${ANDROID_HOME}/platform-tools"
+  # Accept the license.
+  # http://stackoverflow.com/a/17863931/101923
+  sudo apt-get -y install expect
+  expect -c '
+  set timeout -1   ;
+  spawn android update sdk --filter platform-tools,tools,build-tools-23.0.2,android-23,extra-android-support,extra-android-m2repository,extra-google-m2repository --no-ui --force;
+  expect {
+    "Do you accept the license" { exp_send "y\r" ; exp_continue }
+    eof
+  }
+  '
+  # Build and test the app!
+  cd android/CloudVision
+  ./gradlew --console=plain assembleDebug
+  ./gradlew --console=plain lintDebug
+  ./gradlew --console=plain testDebugUnitTest
```

Comment:
I would chain these gradle commands with `&&` - that way if any of them fail (eg return code != 0), that'll be surfaced to the caller of this function